docs/guided-tour/blueprints/helm-chart-resource/README.md

```diff
@@ -3,39 +3,44 @@ title: Helm Chart Component
 sidebar_position: 2
 ---
 
-# Helm Chart Resources in the Component
+# Helm Chart Resources in the Component Version
 
-Let's look at an example in which a Helm Chart is referenced from a Component.
+Let's look at an example in which a Helm Chart is referenced from a component version.
 
 For prerequisites see [here](../../README.md).
 
-## Referencing the Helm Chart without a Component
+## Referencing the Helm Chart without a Component Version
+
 The blueprint of the previous examples reference the Helm chart directly like this:
 
 ```yaml
 chart:
   ref: eu.gcr.io/gardener-project/landscaper/examples/charts/hello-world:1.0.0
 ```
 
-Using this oci reference, the landscaper is able to fetch the helm chart and deploy it correspondingly.
+Using this oci reference, the landscaper is able to fetch the helm chart and deploy it correspondingly. This approach 
+has the disadvantage, that the location of the chart is hidden in the blueprint which is an unknown resource type
+of OCM. Therefore, if OCM component versions are used for transport, scanning etc. such hidden artefact could not be 
+identified as part of the component version and are just skipped. A better approach is to specify such artefacts in 
+the component version itself such that they are known on the component level and just reference the artefacts in 
+the blueprint via their entries in the component descriptor of the component version. We will show this in the 
+following with the helm chart artefact.
```

Comment:
I would remove this part altogether. Just document one approach.
To my understanding this way of referencing charts is deprecated and should not be used.

Reply:
We replace the three example in the blueprints section by one example. In this example all resources (blueprint, chart, image) will be in the component descriptor, the blueprint as local resource. As the other points this is on the follow-up list https://github.tools.sap/kubernetes/k8s-lifecycle-management/issues/1172.